source/mongodb/internal-database-usage.txt

```diff
@@ -4,6 +4,14 @@
 Internal Database Usage
 =======================
 
+.. meta::
+   :description: Learn about how Device Sync interacts with your linked Atlas clusters.
```

Comment:
I don't think this description is right for this page. There are some things on this page that don't seem to be Sync-related - the "System Generated User" and "Transactional Locks" sections seem to be unrelated to Sync.

Reply:
Hmm. Yeah, you're right. I need to rethink this.